account_reconcile_model_strict_match_amount/models/account_reconcile_model.py

```diff
@@ -14,66 +14,50 @@ class AccountReconcileModel(models.Model):
     )
 
     # flake8: noqa
-
     def _get_select_communication_flag(self):
+        comm_flag = super()._get_select_communication_flag()
         if not self.match_total_amount or not self.strict_match_total_amount:
-            return super()._get_select_communication_flag()
-        else:
-            return r"""
-                -- Determine a matching or not with the statement line communication using the aml.name, move.name or move.ref.
-                COALESCE(
-                (
-                    aml.name IS NOT NULL
-                    AND
-                    substring(REGEXP_REPLACE(aml.name, '[^0-9|^\s]', '', 'g'), '\S(?:.*\S)*') != ''
-                    AND
-                        regexp_split_to_array(substring(REGEXP_REPLACE(aml.name, '[^0-9|^\s]', '', 'g'), '\S(?:.*\S)*'),'\s+')
-                        && regexp_split_to_array(substring(REGEXP_REPLACE(st_line.payment_ref, '[^0-9|^\s]', '', 'g'), '\S(?:.*\S)*'), '\s+')
-                )
-                OR
-                    regexp_split_to_array(substring(REGEXP_REPLACE(move.name, '[^0-9|^\s]', '', 'g'), '\S(?:.*\S)*'),'\s+')
-                    &&
-                    regexp_split_to_array(substring(REGEXP_REPLACE(st_line.payment_ref, '[^0-9|^\s]', '', 'g'), '\S(?:.*\S)*'), '\s+')
-                OR
-                (
-                    move.ref IS NOT NULL
-                    AND
-                    substring(REGEXP_REPLACE(move.ref, '[^0-9|^\s]', '', 'g'), '\S(?:.*\S)*') != ''
-                    AND
-                        regexp_split_to_array(substring(REGEXP_REPLACE(move.ref, '[^0-9|^\s]', '', 'g'), '\S(?:.*\S)*'),'\s+')
-                        &&
-                        regexp_split_to_array(substring(REGEXP_REPLACE(st_line.payment_ref, '[^0-9|^\s]', '', 'g'), '\S(?:.*\S)*'), '\s+')
-                )
-                , FALSE)
-                AND
-                CASE
-                    WHEN abs(st_line.amount) < abs(aml.balance) THEN abs(st_line.amount) / abs(aml.balance) * 100
-                    WHEN abs(st_line.amount) > abs(aml.balance) THEN abs(aml.balance) / abs(st_line.amount) * 100
-                    ELSE 100
-                END >= {match_total_amount_param}
-            """.format(
+            return comm_flag
+        # comm_flag.replace is for 13.0
```

Comment:
I'm not sure to understand this comment

Reply:
I think the comment regards https://github.com/OCA/account-reconcile/pull/410/commits/70cf821a4f0526b6320f361a3f6a17a8fb93fd6a#diff-561c565bcdc3ab79d666e97f6dae8f5aee3051c29282c5f0c74c0483d3468a3bR26, which is intended to be ported to 13.0 if needed.